src/styles/main.scss

```
@@ -1,3 +1,45 @@
+$size: 300px;
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+}
+
 body {
   margin: 0;
 }
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
```

```suggestion
  height: 300px;
```
You could create a variable for height